src/index.html

```diff
@@ -11,6 +11,30 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+
+    <form action="http://127.0.0.1:5500/src/index.html" 
+    method="post"
+    class="searchbar"
+    data-qa="big"
+    >
```

Comment:
```suggestion
    <form
      action="http://127.0.0.1:5500/src/index.html" 
      method="post"
      class="searchbar"
      data-qa="big"
    >
```

And it's not good idea to use your local address for action, in this case you can replace it with `'#'`